css/templates/travel/travel.css

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright 2017 The AMP Start Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import './page';
```

Comment:
just import core here and call this page.css

Reply:
Ok re importing core. Regarding calling this ```travel.css```, we are declaring our css-path on the page level in order to keep css size down. Interested to hear your thoughts on this. We figured this template might make an exception since most of the css size is attributed to partials (```templates/travel/partials```) which style components that are page-specific.